### ISSUE TITLE
Add schemas for missing direct marketing metrics

### ISSUE THREAD

**Issue by trieloff** [adobe/xdm#234]
As discussed in https://github.com/adobe/xdm/pull/225#issuecomment-379680193, we need proper schemas for:

* https://ns.adobe.com/xdm/data/metrics/direct-marketing/opens
* https://ns.adobe.com/xdm/data/metrics/direct-marketing/unsubscriptions
* https://ns.adobe.com/xdm/data/metrics/direct-marketing/not-sent

You can take `sends.schema.json` as a starting point.

**Comment by trieloff** [comment]
@saurabhere

**Comment by cdegroot-adobe** [comment]
Why do we need proper schemas for these metrics? There are several that need this rework and I do not understand how they are included and what value the extra complexity brings considering the extra burden.

**Comment by trieloff** [comment]
Every metric needs a proper definition:

- what's the name
- what does it measure
- what unit is used for measurement

The short descriptions we have in the `metrics.schema.json` are not sufficient, because a metric might be used in multiple places.

**Comment by cdegroot-adobe** [comment]
Reopening: Ok, this is a pile of work. I will take it on asap.

**Comment by trieloff** [comment]
We should have definitions for most, and there is the `add-metric.sh` script which can speed up the process.

**Comment by cdegroot-adobe** [comment]
@trieloff Are you ok with the short description names in the metrics.schema.json? assuming they align with the dedicated metric definition?

**Comment by trieloff** [comment]
Yes, you can use these as a starting point. I think all of our metrics can benefit from a rigorous and nuanced definition, but that might take us some rounds of the "what exactly do you mean…" game.
